server/chat/backend/agent/tools/cloud_tools.py

```
@@ -2344,7 +2344,7 @@ def _pinned_trigger(action_id: str = "", _pid=pinned_id, _fn=final_func, **kw):
                      f"Query Bitbucket repositories and files (READ-ONLY during {_ro_reason}). "
                      "Actions: list_repos, get_repo, get_file_contents, get_directory_tree, "
                      "search_code, list_workspaces, get_workspace. "
-                     "Do NOT use create_or_update_file or delete_file."),
+                     "Do NOT use create_or_update_file, edit_file, or delete_file."),
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)